src/Compilers/VisualBasic/Test/Emit/Emit/EditAndContinue/EditAndContinueStateMachineTests.vb

```diff
@@ -976,9 +977,9 @@ End Class
 
         <Fact>
         Public Sub AsyncMethodOverloads()
-            Dim source0 =
-<compilation>
-    <file name="a.vb">
+            Using New EditAndContinueTest().
```

Comment:
Are the changes to this test (and following tests) to rewrite using the current format, or are there behavior changes as well?

Reply:
The new test helpers construct proper syntax map.